thoth/package_extract/image.py

```diff
@@ -515,6 +515,15 @@ def download_image(
     _LOGGER.debug("%s stdout: %s", _SKOPEO_EXEC_PATH, stdout)
 
 
+def get_image_size(path: str) -> int:
```

Comment:
Please add a docstring so coala is happy.

Reply:
> Please add a docstring so coala is happy.

Done!

Reply:
> Also, please check if the computed image size matches what `docker inspect` reports.

@fridex I pulled the same image using `docker pull` and the size I see is 999 MB. But, if I go to https://quay.io/repository/thoth-station/thoth-coala?tab=tags and see the size is 321.9 MB which is closer to what we getting in bytes `337560925`. WDYT?

Reply:
Sounds legit, also see https://github.com/moby/moby/issues/31298